modules/book.js

```diff
@@ -0,0 +1,195 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default class Book {
+  constructor() {
+    this.awesomeBooks = document.querySelector('#awesome-books');
+    this.formSelector = document.querySelector('form');
+    this.titleSelector = document.getElementById('title');
+    this.authorSelector = document.getElementById('author');
+    this.allBtnSelector = this.awesomeBooks.querySelectorAll('.btn');
+
+    this.books = JSON.parse(localStorage.getItem('booksLocalStorage')) || [];
+    this.title = localStorage.getItem('currentTitle') || null;
+    this.author = localStorage.getItem('currentAuthor') || null;
+    this.listSelector = document.getElementById('list-navigation');
+    this.addSeclector = document.getElementById('add-navigation');
+    this.contactSeclector = document.getElementById('contact-navigation');
+    this.navigationSelector = document.getElementById('navigation');
+
+    this.addSection = document.getElementById('added-book');
+    this.formSection = document.getElementById('form-container');
+    this.contactSection = document.getElementById('contact');
+  }
+
+  /* Display an added book to the UI */
+  display() {
+    this.awesomeBooks.innerHTML += `
+  
+      <li id="${this.id}">
+      <h2>"${this.title}"  by   ${this.author}</h2>
+      <button class="btn button-shadow">remove</button>
+      </li>
+  
+      `;
+  }
+
+  /*
+Filter out a removed book from an array of books contianed in each list item
+- add new array of books (with one less book) to the local storage
+*/
+  remove(id) {
+    this.books = this.books.filter((book) => book.id !== id);
+    localStorage.setItem('booksLocalStorage', JSON.stringify(this.books));
+    this.removeBorderFromBooks();
+  }
+
+  /*
+Initialize remove button event listeners
+*/
+  removeDom() {
+    this.allBtnSelector = this.awesomeBooks.querySelectorAll('.btn');
+    this.allBtnSelector.forEach((btn) => {
+      btn.addEventListener('click', (e) => {
+        const parent = e.target.parentNode;
+        this.remove(parent.id);
+        parent.remove();
+      });
+    });
+  }
+
+  /*
+Add a book object to the books array
+- display the book on the UI
+- set booksLocalStorage variable for preserving books array during reload
+- clear currentTitle and currentAuthor variables so that those will
+  not be showing up after a book object has already been added
+- initialize event listeners for the buttons created when adding new books
+*/
+
```

Comment:
- [ optional ]Hi it's not best practice to put comments everywhere in your script. your functions should be self explanatory